crates/aptos/src/lib.rs

```diff
@@ -19,13 +20,15 @@ pub enum Tool {
     Move(move_tool::MoveTool),
     #[clap(subcommand)]
     Op(op::OpTool),
+    List(list::ListResources),
```

Comment:
nit: do we want to maintain alphabetical ordering? I can imagine this list is going to grow very large 😄